portfolio/src/main/webapp/script.js

```diff
@@ -110,3 +111,12 @@ async function deleteAll() {
   fetch("/delete-data", { method: "POST" });
   window.location.reload();
 }
+
+async function loginStatus(){
+  const response = await fetch("/login");
+  const login= await response.text();
+  console.log(login);
+  if(login.includes("true")){
```

Comment:
Doing a string contains on the response is pretty brittle (likely to accidentally break). Instead use `response.sendError` on the server-side to return a status of 401 Unauthorized if the user isn't logged in. Then you can check `response.status` in the JS rather than doing a string contains.